tests/ewasm.rs

```diff
@@ -2294,7 +2294,7 @@ fn selfdestruct() {
     runtime.function("step1", &[]);
     runtime.accounts.get_mut(&runtime.vm.cur).unwrap().1 = 0;
 
-    runtime.function_revert("step2", &[]);
+    runtime.function("step2", &[]);
```

Comment:
Do you know why this changed?

Reply:
We should not remove the `abiDecode` instruction if we don't use the results of a function call, then right?

Reply:
The test could be fixed by changing the to `bool foo = goaway();`.

As far eliding abi decoding, I still think it's a good idea. There are corner cases like this of course.  I am not sure what solc does (and if we care)

Reply:
Fixed